pump/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for pump 1.0
+// Project: https://github.com/mafintosh/pump
+// Definitions by: Tomek Łaziuk <https://github.com/tlaziuk>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+declare function pump(streams: pump.Stream[], callback?: pump.Callback): pump.Stream[];
```

Comment:
Can you add a test which doesn't call `pump` with a callback?
Can you add a test which passes an array of streams?